### PR TITLE
chore: add copyright notices

### PR DESCRIPTION
**Pull request by bdenham**
Added Adobe copyright notices to the source code.

### CODE REVIEW

File: src/AppProvider.jsx [hunk 1]
```diff
@@ -1,3 +1,7 @@
+/* Copyright 2023 Adobe. All Rights Reserved.
+NOTICE: Adobe permits you to use, modify, and distribute this file in
```
Comment (jeff-matthews) [inline src/AppProvider.jsx:2]: Misspelled in all files; do a find/replace

**Review by jeff-matthews — CHANGES_REQUESTED** [review]

File: src/AppProvider.jsx (hunk 1 above)
Comment (jeff-matthews) [reply src/AppProvider.jsx:2]: My eyes were playing tricks on me. Sorry.

**Review by jeff-matthews — APPROVED** [review]

**Comment by bdenham** [comment]
@dshevtsov, good catch on the other PR (adding the correct copyright notice). I added to the same standard notice here.

**Review by dshevtsov — COMMENTED** [review]
I'm not sure about COPYRIGHT and LICENSE. Could you please point me to the corresponding Adobe guidelines?

**Comment by bdenham** [comment]
@dshevtsov, yeah, I have no idea. I'm just including the same COPYRIGHT and LICENSE that appears on all developer.adobe.com sites.

**Comment by dshevtsov** [comment]
If @jeff-matthews is fine with Copiright and License, it's fine with me.

**Review by dshevtsov — APPROVED** [review]

**Pull request merged**